src/utils/narrow.js

```diff
@@ -157,7 +165,7 @@ export const validateNarrow = (narrow: Narrow, streams: Stream[], users: User[])
 };
 
 export const isSameNarrow = (narrow1: Narrow, narrow2: Narrow): boolean =>
-  isEqual(narrow1, narrow2);
+  !narrow1 || !narrow2 ? false : isEqual(narrow1, narrow2);
```

Comment:
That is an overkill.

Reply:
If I remove `!narrow1 || !narrow2` then `expect(isSameNarrow(streamNarrow('stream'), undefined)).toBe(false);` is falling.